x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.tsx

```diff
@@ -259,7 +300,12 @@ export class ESGeoGridSource extends AbstractESAggSource {
     return features;
   }
 
-  _addNonCompositeAggsToSearchSource(searchSource, indexPattern, precision, bufferedExtent) {
+  _addNonCompositeAggsToSearchSource(
+    searchSource: ISearchSource,
+    indexPattern: IndexPattern,
+    precision: number | null,
+    bufferedExtent?: MapExtent | null
```

Comment:
Not sure what the history of this function is, but it appears it's only called from one place and these last two values are always null. Should we just type these two values as `null` only or revise the function to not require these params at all and just always return `null`s where the values were used?

Reply:
discussed offline. There's two callsites for this method. one that sets the params, and one that needs to clear them. Will leave as-is for now.

Reply:
Scratch that. After some offline conversation I found the other usage!